source/deprecations/v3.x.html.md

```diff
@@ -53,3 +53,57 @@ object.notifyPropertyChange('someProperty');
 ##### id: ember-metal.getting-each
 
 Calling `array.get('@each')` is deprecated. `@each` may only be used as dependency key.
+
+### Deprecations Added in Upcoming Versions
+
+#### Use console rather than Ember.Logger
+
+##### until: 4.0.0
+##### id: ember-console.deprecate-logger
+
+The `Ember.Logger` module will be removed at the start of the 4.x cycle. You should remove any calls to `Ember.Logger` and use calls to `console` instead.
+
+In Edge and IE11, uses of `console` beyond calling its methods may require more subtle changes than simply substituting `console` wherever `Logger` appears. In these browsers, they will behave just as they do in other browsers when your development tools window is open. However, when run normally, calls to its methods must not be bound to anything other than the console object or you will receive an `Invalid calling object` exception. This is a known inconsistency with these browsers. (Edge issue #14495220.)
```

Comment:
The various caveats should be under a specific section (e.g. `Migration Gotchas` or somesuch). I see two main gotchas here:

* calling console methods unbound 
* console.debug being missing

FWIW, I think the specific guidance for the latter item there is to just not use console.debug...

Reply:
The section headings at this level aren't terribly distinct, but I broke it up under headings. Looking at a lot of the other deprecations in 3.x and 2.x, the description for this one is a bit more involved, but I think the detail is necessary.